addons/survey/models/survey_question.py

```diff
@@ -49,7 +49,7 @@ class SurveyQuestion(models.Model):
     def default_get(self, fields):
         defaults = super(SurveyQuestion, self).default_get(fields)
         if (not fields or 'question_type' in fields):
-            defaults['question_type'] = False if defaults.get('is_page') == True else 'text_box'
+            defaults['question_type'] = False if defaults.get('is_page') == True else 'simple_choice'
```

Comment:
Runbot ci/style is complaining about this one.
Could remove the "== True" I think.